src/main/java/bio/terra/service/filedata/FSFile.java

```diff
@@ -99,6 +102,24 @@ public FSFile loadTag(String loadTag) {
     return this;
   }
 
+  // TODO: once metadata is collocated with the underlying cloud platform, this can be set
```

Comment:
It would be better to put a JIRA here rather than a TODO, as we don't have a way to track TODOs in our code.

Reply:
I don't know, I feel like I've fixed more TODO's than having unconnected Jiras.  How about we split the difference and I create a Jira but leave the TODO :)